index.html

```diff
@@ -144,5 +144,25 @@ <h3 class="subtitle">Skills</h3>
             </ul>
         </section>
     </section>
+
+    <footer class="contactContainer">
+        <h2 class="title blue">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form class="contactForm" action="https://formspree.io/f/mzbwkvzy" method="POST">
+            <input class="singleLineInput" name="name" type="text" placeholder="Full Name" maxlength="30" required>
+            <input class="singleLineInput" name="email" type="email" placeholder="Email address" required>
+            <textarea class="commentArea" name="comments" placeholder="Enter text here" maxlength="500" required></textarea>
+            <button class="button">
```

Comment:
- [ ] Just as the previous reviewer pointed out, you need to add button interactions which is one of the things that makes a button to be alive to the user. Interactions such as hover and co.

Reply:
@Olamarx Well it has been implemented so what else do I need to add. I believe I followed the interactions that the template in Figma shows. Please explain in a bit more detail what I need to add. Thanks.

Default state of the button
![image](https://user-images.githubusercontent.com/114885671/199368285-739a537f-ef11-4873-b980-84c20be1e4c4.png)

The button is pressed with all the required fields.
![image](https://user-images.githubusercontent.com/114885671/199368353-03827bd5-fbbc-4ff0-93d7-295826c9a0c1.png)

This is the code where it was added before and this button inherits the behavior.
![image](https://user-images.githubusercontent.com/114885671/199368407-c56555ac-e48d-4564-b2ab-4e66f8bb60c0.png)

Reply:
You only need to add the hover m. You have added the active but the over is not yet added. Just duplicate the active and change active to hover. Then you are good to go.